app/src/main/java/org/mascota/ui/view/diary/DiaryDetailWriteFragment.kt

```diff
@@ -0,0 +1,229 @@
+package org.mascota.ui.view.diary
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Build
+import android.provider.MediaStore
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.Log
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.annotation.RequiresApi
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import org.mascota.R
+import org.mascota.databinding.FragmentDiaryDetailWriteBinding
+import org.mascota.ui.base.BindingFragment
+import org.mascota.ui.view.diary.adpter.SpinnerAdapter
+import org.mascota.ui.view.diary.data.SpinnerModel
+import java.time.LocalDate
+import java.time.LocalDate.now
+import java.time.format.DateTimeFormatter
+
+
+@Suppress("DEPRECATION")
+class DiaryDetailWriteFragment : BindingFragment<FragmentDiaryDetailWriteBinding>(R.layout.fragment_diary_detail_write){
+
+    val REQ_STORAGE_PERMISSION: Int = 1
+    val REQ_GALLERY = 12
+    @RequiresApi(Build.VERSION_CODES.O)
+    val current_date : LocalDate = now()
+    @RequiresApi(Build.VERSION_CODES.O)
+    val formatter = DateTimeFormatter.ofPattern("yyyy.MM.dd")
+    @RequiresApi(Build.VERSION_CODES.O)
+    val tv_today = current_date.format(formatter).toString()
+
+    private lateinit var spinnerAdapter: SpinnerAdapter
+    private var item_spin = arrayListOf<SpinnerModel>()
+
+    val image = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+
+            activityResult ->
+        if(activityResult.resultCode == Activity.RESULT_OK){
+            val intent = activityResult.data
+            if(intent != null){
+                val fileUri = intent.data
+                binding.iv1.setImageURI(fileUri)
+            }
+            else{
+                Log.d("reusltcode","못함")
+            }
+        }
+
+    }
+
+
+
+    override fun initView() {
+        setupSpinner()
+        initClickSelectImage()
+        binding.tvToday.setText(tv_today)
+        WriteTitle()
+
+
+
+    }
+
+    private fun initList(){
+        item_spin.add(SpinnerModel(tv_chapter = "제 1장",tv_chatitle = "코봉이의 적응"))
+        item_spin.add(SpinnerModel(tv_chapter = "제 2장", tv_chatitle = "코봉아 아프지마"))
+        item_spin.add(SpinnerModel(tv_chapter = "제 3장", tv_chatitle = "코봉아 사랑해"))
+
+
+    }
+
+    private fun setupSpinner(){
+
+       initList()
+
+        spinnerAdapter = context?.let { SpinnerAdapter(it, R.layout.item_spinner, item_spin) }!!
```

Comment:
이렇게 작성하는 것 보다, requireContext()를 활용해, Not-Null한 콘텍스트를 받아오는게 훨씬 깔끔합니당..! 대충
```suggestion
        spinnerAdapter = SpinnerAdapter(requireContext(), R.layout.item_spinner, item_spin)
```
이런식으로요!!